src/pages/login/login.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import Image from 'next/image';
+import { Inter } from "next/font/google";
+import styles from "./login.module.css";
+import CustomizableAside from "../../components/customizableAside";
+
+
+const { useState } = React;
+
+function LoginForm() {
+
+    const [values, setValues] = useState({
+        email: "",
+        password: "",
+    });
+
+    const [error, setError] = useState(undefined);
+    const handleSubmit = async (e) => {
+        e.preventDefault();
+
+        try {
+            const response = await axios.post("/login", values);
+            console.log(response.data);
+            // handle success.
+        } catch (error) {
+            console.error(error);
+            // handle error (example).
+            setError("An error occurred, try again...");
+        }
+        
+
+    }
+    
+    return (
+        <div className={styles.loginPage}>
+
+            <div className={styles.asideContainer}>
+                <CustomizableAside />
+            </div>
+
+            <div className={styles.loginContainer}>
+                    <div className={styles.loginImage} >
+                        <Image src="/images/customizableAvatar.svg" width="70" height="70" priority  alt=''/>
+                    </div>
+                    <form onSubmit={handleSubmit} className={styles.form}>
+                        <div className={styles.field}>
+                            <label htmlFor="email" className={styles.formLabel}>Email Address</label>
+                            <input
+                            className={styles.input}
+                            onChange={(e) => setValues({ ...values, email: e.target.value })}
+                            value={values.email}
+                            type="text"
+                            name="email"
+                            placeholder="joe.deo@gmail.com"
+                            />
+                        </div>
+                        <div className= {styles.field}>
+                            <label htmlFor="password" className={styles.formLabel}>Password</label>
+                            <input
+                            className={styles.input}
+                            onChange={(e) => setValues({ ...values, password: e.target.value })}
+                            value={values.password}
+                            type="password"
+                            name="password"
+                            placeholder="*********"
+                            />
+                            <p className={styles.que}>Forgot Password?</p>
+                        </div>
+                        {error ? <p className= {styles.error}>{error}</p> : <></>}
```

Comment:
@Sheyitofunmi Do you think we could render the logic inside the `<p>` tag instead of rendering a fragment if `error` is undefined?

```js
<p className= {styles.error}>
   {error ? error : ""}
</p>
```

Maybe because error is `undefined` by default, my approach creates a problem? @jjleigh please weigh in. 🙏🏾